tsconfig.json

```
@@ -13,7 +13,11 @@
     "moduleResolution": "node",
     "resolveJsonModule": true,
     "isolatedModules": true,
-    "jsx": "preserve"
+    "jsx": "preserve",
+    "baseUrl": ".",
+    "paths": {
+      "@*": ["./*"]
```

Needs the slash to prevent if from doing e.g. `@chainflip/sdk` to `./chainflip/sdk`. or if we don't want to use the slash here, we have to revert the change to manually alias all directories
```suggestion
      "@/*": ["./*"]
```